api/middleware.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/vocdoni/vocdoni-z-sandbox/log"
 )
 
+// DisabledLogging is a global flag to disable logging middleware
+var DisabledLogging = false
```

Comment:
Why not declare another logger only for the API with a single level for it?
(I dont know even if it is possible with our current logger, but I think that is better than use this boolean in the whole package)